client/my-sites/patterns/bodies/home/index.tsx

```diff
@@ -54,7 +37,6 @@ export const PatternsHomePage = ( {
 					link: `/patterns/${ category.name }`,
 				} ) ) }
 			/>
-			<PatternGallery patterns={ patterns } isGridView={ isGridView } />
```

Comment:
Why the list of regular patterns was removed?

Reply:
It's moved to `pages/category/index.tsx`. It was here temporarily before I created a page for it.